tests/framework/db/sqlite/conditions/InConditionBuilderTest.php

```
@@ -0,0 +1,89 @@
+        self::assertSame(
+            $expectedParams,
+            $params,
+            'Bound parameters do not match expected parameters.',
+        );
+    }
+
+    public function testThrowNotSupportedExceptionWhenBuildSubqueryInConditionIsCompositeColumns(): void
+    {
+        $query = new Query();
+
+        $db = $this->getConnection(true, false);
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $db. Configured minimum length is 3. (undefined)

(ShortVariable)